atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -509,7 +509,8 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
-                    'config': config
+                    'config': config,
+                    'autorebuild': is_rebuild(self.workflow)
```

Comment:
@mikebonnet, @mikem23: any preference as to the key that should be used in content-generator build's metadata to indicate it was automatically rebuilt?